tests/v2_tests/test_hybrid_search.py

```diff
@@ -231,3 +231,40 @@ def test_hybrid_search_with_filter(self):
 
                     self.assertEqual(len(hybrid_res["hits"]), 1)
                     self.assertEqual(hybrid_res["hits"][0]["_id"], "doc8")
+
+    def test_hybrid_search_rrf_with_replicas_has_no_duplicates(self):
+        """
+        Tests that show that running 100 searches on indexes with 3 replicas (structured text & unstructured text)
+        will not have duplicates in results.
+        Only relevant for cloud tests.
+        """
+
+        if not self.client.config.is_marqo_cloud:
+            self.skipTest("Test is not relevant for non-Marqo Cloud instances")
+
+        index_test_cases = [CloudTestIndex.structured_text, CloudTestIndex.unstructured_text]
+        for cloud_test_index_to_use in index_test_cases:
+            test_index_name = self.get_test_index_name(
+                cloud_test_index_to_use=cloud_test_index_to_use,
+                open_source_test_index_name=None
```

Comment:
We are not running this test in the open-source right?

Reply:
we're not, but `open_source_test_index_name` is a required parameter, so I set it to `None`. This test should ignore itself for open source